distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -85,8 +87,8 @@ org.codice.ddf.system.registry-id=
 # For Solr Server:
 solr.client=HttpSolrClient
 solr.http.port=8994
-# Use "https" in the URL to configure the Solr server for TLS
-solr.http.url=http://localhost:_DO_NOT_EXPAND_${solr.http.port}/solr
+solr.http.protocol=https
```

Comment:
Added this property so that the scripts can decide to start Solr with or without TLS